tools/Azure.Mcp.Tools.ConfidentialLedger/tests/Azure.Mcp.Tools.ConfidentialLedger.LiveTests/ConfidentialLedgerCommandTests.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Mcp.Tests;
+using Azure.Mcp.Tests.Client;
+using Xunit;
+
+namespace Azure.Mcp.Tools.ConfidentialLedger.LiveTests;
+
+public class ConfidentialLedgerCommandTests(ITestOutputHelper output) : CommandTestsBase(output)
+{
+    [Fact]
+    public async Task Should_append_entry_successfully()
+    {
+        // Arrange
+        var ledgerName = Settings.DeploymentOutputs["CONFIDENTIAL_LEDGER_NAME"];
+        var testContent = $$"""
+            {
+                "message": "Live test entry",
+                "timestamp": "{{DateTime.UtcNow:o}}",
+                "testRun": "{{Guid.NewGuid()}}"
+            }
+            """;
+
+        // Act
+        var result = await CallToolAsync(
+            "azmcp_confidentialledger_entries_append",
+            new()
+            {
+                { "ledger", ledgerName },
+                { "content", testContent }
```

Comment:
Should be `entry-data`, or to flip this if wanted, rename `entry-data` to `content`